db/src/api/satapi.sql

```diff
@@ -64,16 +77,21 @@ SELECT
     WHERE (
       g.geom IS NULL OR 
       data.ST_Intersects(g.geom, c.geom)
-    );
-$$ LANGUAGE SQL IMMUTABLE;
-
-CREATE FUNCTION searchnogeom(
-  include TEXT[] = NULL
-)
-RETURNS setof api.collectionitems
-AS $$
-SELECT * FROM search(NULL, NULL, include);
-$$ LANGUAGE SQL IMMUTABLE;
+    )'
+||
```

Comment:
Hey @sharkinsspatial, this looks like a really cool project!

Looks like your doing dynamic sql here. Just wanted to chime in and suggest using the pg [format](https://www.postgresql.org/docs/current/plpgsql-statements.html#PLPGSQL-QUOTE-LITERAL-EXAMPLE) function. An example for using DISTINCT on postgrest:

```sql
create function projects(dist name) returns setof projects as $_$
begin
  return query execute format($$
    select distinct on(%I) * from projects
  $$, dist);
end; $_$ language plpgsql;
```

As you can see highlighting works and you can even go multi-line. It's also "safer" because it's harder to forget doing a `quote_ident`/`quote_literal`.